examples/nsl_pk/DY.Example.NSL.Protocol.Stateful.fst

```diff
@@ -72,7 +76,7 @@ let send_msg1 global_sess_id alice sess_id =
   match st with
   | InitiatorSentMsg1 bob n_a -> (
     let*? pk_b = get_public_key alice global_sess_id.pki (PkEnc "NSL.PublicKey") bob in
-    let* nonce = mk_rand PkNonce (principal_label alice) 32 in
+    let* nonce = mk_rand PkNonce (principal_tag_label alice "DY.Lib.State.PrivateKeys") 32 in
```

Comment:
Do we really gain something from making the labels more complicated in the NSL example? When I started with this DY* version, I liked that it was relatively simple, but now I think we are making it much harder for beginners to understand such a simple example.

Reply:
For every label I defined a nice name for this label, making the code less intimidating (especially with the long `"DY.Lib.State.PrivateKeys"`).
I also simplified the label of NSL nonces to a simple `principal_label` instead of the full-fledged `join (long_term_key_label ...) (principal_tag_label ...)` (although this could be reverted with minimal cost now that we define labels at the top of the Stateful file)